src/coprocessor/dag/expr/scalar_function.rs

```diff
@@ -1159,6 +1161,9 @@ mod test {
                     ScalarFuncSig::JsonUnquoteSig,
                     ScalarFuncSig::ASCII,
                     ScalarFuncSig::Bin,
+                    ScalarFuncSig::BitNegSig,
```

Comment:
Why is this line added? Rest LGTM.

Reply:
@huachaohuang  sorry, the line is deleted.